src/plugins/data/server/search/strategies/eql_search/eql_search_strategy.ts

```diff
@@ -45,11 +46,11 @@ export const eqlSearchStrategyProvider = (
           uiSettingsClient
         );
         const params = id
-          ? getDefaultAsyncGetParams(null, options)
+          ? getCommonDefaultAsyncGetParams(null, options)
           : {
               ...(await getIgnoreThrottled(uiSettingsClient)),
               ...defaultParams,
-              ...getDefaultAsyncGetParams(null, options),
```

Comment:
This looks strange to me that this uses `Get` async helpers instead of `Submit`, 
Created an issue for EQL clean up and align: https://github.com/elastic/kibana/issues/128635